Go instrumentation tool: More README edits (#49)
newrelic/go-easy-instrumentation: README.md

<a href="https://opensource.newrelic.com/oss-category/#new-relic-experimental"><picture><source media="(prefers-color-scheme: dark)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/dark/Experimental.png"><source media="(prefers-color-scheme: light)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Experimental.png"><img alt="New Relic Open Source experimental project banner." src="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Experimental.png"></picture></a>

> # Go instrumentation tool

Reviewer: ```suggestion
# Go Easy Instrumentation [![codecov](https://codecov.io/gh/newrelic/go-easy-instrumentation/graph/badge.svg?token=0qFy6WGpL8)](https://codecov.io/gh/newrelic/go-easy-instrumentation)
```

We want `Go Easy Instrumentation` to be the name of the product for now, and I would like the `codecov` badge displayed to show that this is a dependable product.

Author: @iamemilio on the main tech docs site, I just pushed up the label "Go easy instrumentation tool" with lower case to follow our style guide. Can we do that here, too?

I'm not sure how to set the code coverage to "dependable." It looks like that would require a custom badge. Can you figure that out?

Reviewer: Hi, yes. You can just restore the badge as it was: `[![codecov](https://codecov.io/gh/newrelic/go-easy-instrumentation/graph/badge.svg?token=0qFy6WGpL8)](https://codecov.io/gh/newrelic/go-easy-instrumentation)`

Having coverage above 80% is considered strong coverage.

Before you start the installation steps below, make sure you have a version of Go installed that is within the support window for the current [Go programming language lifecycle](https://endoflife.date/go).

1. Clone this repository to a directory on your system. For example:
    ```sh

>     git clone .../go-agent-pre-instrumentation

Author: ```suggestion
    git clone .../go-easy-instrumentation.git
```

    ```
2. Go into that directory:
    ```sh

>     cd go-agent-pre-instrumentation

Author: ```suggestion
    cd go-easy-instrumentation
```

    ```
3. Resolve any third-party dependencies:
    ```sh
    go mod tidy
    ```
## Generate instrumentation suggestions

For detailed instructions on how to generate instrumentation suggestions, see our documentation at [docs.newrelic.com](https://docs/apm/agents/go-agent/installation/install-automation-new-relic-go/#generate-suggestions).

## Support

> This is an experimental product, and New Relic is not offering official support at the moment. Please create issues in Github if you are encountering a problem that you're unable to resolve. When creating issues, its vital to include as much of the prompted for information as possible. This enables us to get to the root cause of the issue much more quickly. Please also make sure to search existing issues before creating a new one.

Author: ```suggestion
This is an experimental product, and New Relic is not offering official support at the moment. Please create issues in Github if you are encountering a problem that you're unable to resolve. When creating issues, its vital that you include as much of the requested information as possible. This enables us to get to the root cause of the issue much more quickly. Please also make sure to search existing issues before creating a new one.
```

## Contributing

> We encourage your contributions to improve the Go instrumentation tool! Keep in mind when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You only have to sign the CLA one time per project.

Reviewer: ```suggestion
We encourage your contributions! Keep in mind when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You only have to sign the CLA one time per project.
```

If you have any questions, or to execute our corporate CLA, required if your contribution is on behalf of a company, please drop us an email at opensource@newrelic.com.

## License

> This Go instrumentation tool is licensed under the [Apache 2.0](http://apache.org/licenses/LICENSE-2.0.txt) License.

Reviewer: ```suggestion
Go easy instrumentation is licensed under the [Apache 2.0](http://apache.org/licenses/LICENSE-2.0.txt) License.
```

> >The Go instrumentation tool also uses source code from third-party libraries. You can find full details on which libraries are used and the terms under which they are licensed in the third-party notices document.

Reviewer: ```suggestion
>This tool also uses source code from third-party libraries. You can find full details on which libraries are used and the terms under which they are licensed in the third-party notices document.
```
```suggestion
>This tool also uses source code from third-party libraries. You can find full details on which libraries are used and the terms under which they are licensed in the third-party notices document.
```
